src/System.Management.Automation/engine/PropertyAccessor.cs

```diff
@@ -31,13 +30,9 @@ internal abstract class ConfigPropertyAccessor
         /// </summary>
         static ConfigPropertyAccessor()
         {
-#if CORECLR
             Instance = Platform.IsInbox
```

Comment:
This can be simplified to:
Instance = new JsonConfigFileAccessor(); 

Platform.IsInbox() should be removed from the code as well